tools/releaseBuild/azureDevOps/templates/compliance.yml

```diff
@@ -79,12 +79,11 @@ jobs:
     inputs:
       targetType: F
       optionsFC: 0
-      optionsXS: 0
+      optionsXS: 1
```

Comment:
This is not what is required for recursion. I believe that we need to set `targetArgument` instead.

Reply:
To clarify, this option _is_ required, but it is the default. PowerShell needs to not override it.

Reply:
I'm seeing we do not use the Compliance repo, otherwise https://github.com/PowerShell/Compliance/pull/28 would have fixed this.

Reply:
Opened https://github.com/TravisEz13/PowerShell/pull/144

Reply:
reopened as https://github.com/PowerShell/PowerShell/pull/16035